apps/web/pages/api/cron/calendar-cache-cleanup.ts

```diff
@@ -3,6 +3,12 @@ import type { NextApiRequest, NextApiResponse } from "next";
 import prisma from "@calcom/prisma";
 
 export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const apiKey = req.headers.authorization || req.query.apiKey;
+  if (![process.env.CRON_API_KEY, `Bearer ${process.env.CRON_SECRET}`].includes(`${apiKey}`)) {
+    res.status(401).json({ message: "Not authenticated" });
+    return;
+  }
+
```

Comment:
`nitpick`: For readability, would be good to abstract into it's own function.

```suggestion

const isValidKey = (apiKey: string) => {
  return [process.env.CRON_API_KEY, `Bearer ${process.env.CRON_SECRET}`].includes(apiKey);
}

export default async function handler(req: NextApiRequest, res: NextApiResponse) {
  const apiKey = req.headers.authorization || req.query.apiKey;
  if (!isValidKey(apiKey)) {
    res.status(401).json({ message: "Not authenticated" });
    return;
  }
```